packages/report-server/src/aggregator/Aggregator.ts

```diff
@@ -1,56 +1,38 @@
 import { Aggregator as BaseAggregator } from '@tupaia/aggregator';
-import { QueryBuilder } from './QueryBuilder';
-import { buildAggregationOptions } from './buildAggregationOptions';
+import { getDefaultPeriod } from '@tupaia/utils';
+import { Event } from '../types';
 
 export class Aggregator extends BaseAggregator {
-  constructor(dataBroker, routeHandler) {
-    super(dataBroker);
-    this.routeHandler = routeHandler;
-  }
-
   async fetchAnalytics(
-    dataElementCodes,
-    originalQuery,
-    replacementValues,
-    initialAggregationOptions = {},
+    dataElementCodes: string[],
+    organisationUnitCodes: string,
+    period = getDefaultPeriod(),
```

Comment:
`period` is a DHIS specific concept, I believe we support just `startDate` and `endDate` in our internal data fetching (see https://github.com/beyondessential/tupaia/blob/dev/packages/data-api/src/fetchData.js#L39). I see us using `period` less and less in the future

But you can leave it as such for now and come back at a later stage to update it

Reply:
I think I may leave it for now if that's okay, and we can redesign this when we are actually implementing some reports. `startDate` and `endDate` are more effecient and sensible, so I'm happy with going with those in the future

Reply:
Good call!